deps/chakrashim/src/v8function.cc

```diff
@@ -87,7 +87,7 @@ Local<Value> Function::Call(Handle<Value> recv,
 }
 
 void Function::SetName(Handle<String> name) {
-  JsErrorCode error = DefineProperty((JsValueRef)this,
+  DefineProperty((JsValueRef)this,
```

Comment:
Same as before, "error" was only referenced in a debug assert and not used in Release build, compiler complains.